src/main/java/org/broadinstitute/hellbender/tools/walkers/ReferenceConfidenceVariantContextMerger.java

```diff
@@ -368,8 +372,9 @@ private static void removeStaleAttributesAfterMerge(final Map<String, Object> at
                     values.add(Integer.parseInt(stringValue));
                 }
             } catch (final NumberFormatException e) {
-                // nothing to do
-                //TODO This comes directly from gatk3, we should determine if we should log this or if it happens so much it is not useful to log
+                final String baseMsg = "Remaining (non-reducible) annotations are assumed to be ints or doubles, but " + value + " doesn't parse and will not be annotated in the final VC.";
+                final String msg = value.toString().contains("|") ? baseMsg + " Add -G Standard -G AS_Standard to the command to annotate in the final VC." : baseMsg;
```

Comment:
From @droazen This seems like a pretty brittle way of detecting whether AS annotations are present...